lib/tether/ops_linux.go

```diff
@@ -661,13 +663,32 @@ func (t *BaseOperations) MountLabel(ctx context.Context, label, target string) e
 	}
 
 	if err := Sys.Syscall.Mount(label, target, "ext4", syscall.MS_NOATIME, ""); err != nil {
+		// consistent with MountFileSystem
 		detail := fmt.Sprintf("mounting %s on %s failed: %s", label, target, err)
 		return errors.New(detail)
 	}
 
 	return nil
 }
 
+// MountFileSystem performs a mount based on the target path from the source url
+// This assumes that the source url is valid and available.
+func (t *BaseOperations) MountFileSystem(ctx context.Context, source url.URL, target string, mountOptions string) error {
+	defer trace.End(trace.Begin(fmt.Sprintf("Mounting %s on %s", source.String(), target)))
+
+	if err := os.MkdirAll(target, 0600); err != nil {
```

Comment:
Assuming this reply was for the review comment above.  If so, then Mountlabel sounds fine since you're mounting a filesystem by a filesystem label.